internal/generated/snippets/speech/apiv1/Client/CreateCustomClass2/main.go

```diff
@@ -0,0 +1,71 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Code generated by cloud.google.com/go/internal/gapicgen/gensnippets. DO NOT EDIT.
+
+// [START speech_v1_generated_Adaptation_CreateCustomClass_sync_2]
+
+package main
+
+import (
+	"context"
+	"log"
+
+	speech "cloud.google.com/go/speech/apiv1"
+	"google.golang.org/api/option"
+	speechpb "google.golang.org/genproto/googleapis/cloud/speech/v1"
+)
+
+func main() {
+	ctx := context.Background()
+	c, err := speech.NewAdaptationClient(ctx,
+		option.WithEndpoint("us-speech.googleapis.com:443"))
+	if err != nil {
+		log.Fatal(err)
+	}
+	defer c.Close()
+	// The custom class parent element
+	parent := "projects/[PROJECT]/locations/us"
+	// The id for the custom class
+	customClassId := "passengerships"
```

Comment:
These two should have been parameters of the sample method, so that it is testable. Not sure if you can add these to the `main` function in go or if you need to define a method with the parameters and then call that from the `main` function with the config specified values? I'll leave that decision to you and Go folks as you know what's most idiomatic.

Can I please ask you to update this code with whatever solution you find and let me know. Thanks!

Reply:
Thanks. We will need to define a method with the parameters and then call that from the `main` function.

Reply:
Fixed in [speech/apiv1/AdaptationClient/create_custom_class.go](https://github.com/googleapis/google-cloud-go/pull/6688/files#diff-ee0271242537214235b0f2e258beb58df03a416814810cd805bb70a8b622438b).